code/mob/living/intangible/flock/flockmind.dm

```diff
@@ -58,8 +58,21 @@
 /mob/living/intangible/flock/flockmind/Life(datum/controller/process/mobs/parent)
 	if (..(parent))
 		return TRUE
+	if (!src.flock)
+		return
 	src.flock.peak_compute = max(src.flock.peak_compute, src.flock.total_compute())
-	if (src.started && src.flock)
+	if (get_turf(src) == src.previous_turf)
+		src.afk_counter++
+		if (src.afk_counter > round(src.afk_counter_threshold / parent.schedule_interval))
```

Comment:
Not really a big issue, but this is the wrong way of mult adjusted counting. Since `schedule_interval` can change over the course of the round, you should add it to the counter each time instead of dividing at the end.